auth/authenticate_implicit_with_adc.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package snippets
+
+// [START auth_cloud_implicit_adc]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"cloud.google.com/go/storage"
+	"google.golang.org/api/iterator"
+)
+
+// authenticateImplicitWithAdc auto-detects credentials
+// and prints storage buckets by authenticating with ADC.
```

Comment:
Maybe state the full-length term Application Default Credentials, or else provide a link?

Reply:
+1, maybe just
authenticateImplicitWithAdc uses Application Default Credentials
to automatically find credentials and authenticate.